articles/user-interface-animation-with-anime.js/index.md

```diff
@@ -0,0 +1,566 @@
+```
+
+We can also use the [Node Package Manager (npm)](https://www.npmjs.com/).
+
+```bash
+$ npm install animejs --save
+```
+
+Another alternative is to use the latest release of the library hosted on a CDN. See the code below.
+
+```html
+<script src="https://cdnjs.cloudflare.com/ajax/libs/animejs/3.2.1/anime.min.js"></script>
+```
+
+### Getting started with Anime.js
+
+We use the `anime()` function to create the animations. The `anime()` function takes an object as an argument. The object holds the animation details. You can refresh your knowledge on [Javascript Objects](https://www.w3schools.com/js/js_object_definition.asp).
+
+```javascript
+let animation = anime({
+  /* animation details here*/
+});
+```
+
+or
+
+```javascript
+anime({
+  /* animation details here*/
+});
+```
+
+In the animation details, we need to define targets, properties, property parameters, and animation parameters.
+
+1. **Target:** This refers to the element to be animated. It can be any CSS selector, DOM Node, NodeList, JavaScript Object, or an array containing multiple targets. Examples are `div`, `.classSelector`, and `#idSelector`. Target tell anime how to find the elements on the web page.
+
+2. **Properties:** This refers to the CSS Properties that can be animated, CSS Transforms, Object Properties, Dom Attributes, and SVG Attributes. They include `translate`, `rotate`, `scale`, `skew`, `opacity`, `color`, among others.
+
+3. **Property parameters:** `duration`, `delay`, `endDelay`, `easing`, `round`, e.t.c
+
+4. **Animation Parameters:** These are Direction, Loop, and Autoplay. Refer to [Anime.js Documentation](https://animejs.com/documentation/#direction) for more details.
+
+In the next section, we will implement the above. Follow the comments in the code.
+
+### Simple Animations with Anime.js
+
+#### Shapes Animation
+
+In this example, we will create a triangular shape inside a `div` identified by `#triangle1`. You require basic CSS knowledge to achieve this.
+**Javascript code:**
+
+```javascript
+  <script type="text/javascript">
+    //Create Anime object
+    anime({
+      //target
+      targets: "#triangle1",
+      //Properties
+      rotateY: 360,
+	  scale: 0.5,
+	  translateX: 300,
+	  skew: 60,
+      // Property Parameters
+      duration: 1000,
+      endDelay: 300,
+      easing: "easeInOutSine",
+	  // Animation Parameters
+	  direction: 'alternate',
+      loop: true,
+    });
+  </script>
+```
+
+**Explanation**
+In the javascript code example above the target is `#triangle1`. Four transformations are applied to the target all at once. The resulting animation is shown in the output.
+`rotateY: 360`: Rotates the target along the Y-axis for 360&deg;
+`scale: 0.5`: Scales the target by a factor of 0.5.
+`translateX: 300`: Moves the target 300 pixels along the X-axis.
+`skew: 60`: Skews the target 60&deg;.
+Other properties used in Anime.js can be found on [Anime.js documentation](https://animejs.com/documentation/#CSStransforms).
```

Comment:
I'm not exactly sure what you wanted this to look like, but it looks weird. In Markdown, a single newline doesn't create a new paragraph. I'm pretty sure you wanted the "Explanation" header to be its own line, so you'll need to add another newline. There are several other Explanation headers you'd need to fix too. I think it would make sense for the property explanations to be bullet points, in which case you need to do something like this:

```markdown
- `rotateY: 360`: Rotates the ...
- `scale: 0.5`: Scales the target ...
...
```
It also makes sense that the last line should be its own line. You'll need a newline there too. Look at the rendered Markdown file in GitHub to check that it's formatted the way you want.